js/wp-seo-featured-image-302.js

```diff
@@ -60,6 +59,37 @@
 		return content;
 	};
 
+	/**
+	 * Check if image is smaller than 200x200 pixels. If this is the case, show a warning
+	 * @param {object} featuredImage
+	 */
+	function checkFeaturedImage( featuredImage ) {
+		var attachment = featuredImage.state().get( 'selection' ).first().toJSON();
+
+		if ( attachment.width < 200 || attachment.height < 200 ) {
+			//Show warning to user and do not add image to OG
+			if ( 0 === $( '#yst_opengraph_image_warning' ).length ) {
+				$( '<div id="yst_opengraph_image_warning"><p>' + wpseoFeaturedImageL10n.featured_image_notice + '</p></div>' ).insertBefore( '#postimagediv' );
+				var $postImageDiv = $( '#postimagediv' );
```

Comment:
If you move this line up and use it when inserting you prevent a duplicate jQuery selector DOM call.